cpp/src/gandiva/precompiled/string_ops_test.cc

```diff
@@ -912,6 +912,35 @@ TEST(TestStringOps, TestReverse) {
   ctx.Reset();
 }
 
+TEST(TestStringOps, TestLevenshtein) {
+  gandiva::ExecutionContext ctx;
+  uint64_t ctx_ptr = reinterpret_cast<gdv_int64>(&ctx);
+
+  EXPECT_EQ(levenshtein_utf8_utf8(ctx_ptr, "kitten", 6, "sitting", 7), 3);
```

Comment:
Can you add more test cases from the wikipedia page? or even from the Hive UDF implementation?

The logic is quite complex, it is better to have good test coverage